themes/digital.gov/layouts/authors/list.html

```diff
@@ -93,9 +93,9 @@ <h1>{{- .Params.display_name | markdownify -}}</h1>
                     {{- if .Params.twitter -}}
                       <li>
                         <a
-                          class="twitter"
+                          class="x"
                           rel="external"
-                          title="{{- .Params.display_name -}} on Twitter"
+                          title="{{- .Params.display_name }} on X"
                           href="http://www.twitter.com/{{- .Params.twitter -}}/"
```

Comment:
@clmedders @nick-mon1 confirming, we're leaving the domain in `href` the same; i.e. `http://www.twitter.com/`? Either way we should use `https` for security.

Reply:
updated to https: